### PR TITLE
Fixes slowdown applying on open spaces

### PR DESCRIPTION
**Pull request by Alberyk**
With this change you won't be slowed anymore while floating on an open space, such as using a jetpack.

### CODE REVIEW

**Comment by Lohikar** [comment]
One problem; won't this also apply to openturfs with lattices/catwalks?

**Comment by Alberyk** [comment]
Same behavior happened with space, so, I see little issue with it.

**Comment by Lohikar** [comment]
I mean, it makes sense on space (no gravity). Many openturfs do have gravity.

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 	if (istype(loc, /turf/space)) return -1 // It's hard to be slowed down in space by... anything
 
+	if (istype(loc, /turf/simulated/open)) //Open space things
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/human_movement.dm]: We have a helper for this. `isopenspace(loc)` or somesuch. Use that!

Comment (skull132) [inline code/modules/mob/living/carbon/human/human_movement.dm]: We have a helper for this. isopenspace(loc) or somesuch. Use that!

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 2]
```diff
@@ -6,6 +6,11 @@
 
 	if (istype(loc, /turf/space)) return -1 // It's hard to be slowed down in space by... anything
 
+	if (istype(loc, /turf/simulated/open)) //Open space things
+		if(!(locate(/obj/structure/lattice, loc) || locate(/obj/structure/stairs, loc) || locate(/obj/structure/ladder, loc)))
+			if(!has_gravity(src, loc))
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/human_movement.dm]: Move this to the clause at line 9. It's cheaper to check for this, than it is to use the locate calls.

`if (isopenturf(loc) && !has_gravity(src, loc))`

**Pull request merged**